e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/SchedulingTests.java

```diff
@@ -274,11 +274,12 @@ void schedulingCoexistenceGoal() throws IOException {
   void schedulingCoexistenceGoalWithAnchor() throws IOException {
```

Comment:
these fixes could be squashed with the other changes you made previously to `SchedulingTests`.